test/src/derive.rs

```diff
@@ -1,5 +1,11 @@
 // Licensed under the Apache-2.0 license
 
+// The IV fed to the DOE when the ROM deobfuscates the UDS seed (as passed to doe registers)
+pub const DOE_UDS_IV: [u32; 4] = [0xfb10365b, 0xa1179741, 0xfba193a1, 0x0f406d7e];
+
+// The IV fed to the DOE when the ROM deobfuscates the field entropy seed (as passed to doe registers)
```

Comment:
Uber nit: no need to refer to this as "field entropy seed". UDS seed was chosen for DICE compliance reasons IIRC.

Reply:
This code deals with both the obfuscated field entropy and the deobfuscated field entropy. I've been calling the deobfuscated value the "field entropy" and the obfuscated the "field entropy seed", to match the UDS stuff. What are the correct terms to use to distinctly refer to these two values?

Reply:
Ah, I suppose "field entropy seed" is alright then.